### ISSUE TITLE
Add an "Email" field to the Insert Link modal

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/cleanslate-cms#54]
_From @adamjohnson on July 24, 2014 16:11_

I have had two people *today* have issues with linking email addresses in CleanSlate. They've tried to use the URL field with an email and do this:

![https://s3.amazonaws.com/f.cl.ly/items/451L1g0o0p2U2D35160z/cleanslate-email-link-modal.png](https://s3.amazonaws.com/f.cl.ly/items/451L1g0o0p2U2D35160z/cleanslate-email-link-modal.png)

Which outputs this:

```html
<a href="firstname.lastname@mail.wvu.edu">firstname.lastname@mail.wvu.edu</a>
```

Note the missing `mailto:`:

![https://s3.amazonaws.com/f.cl.ly/items/430H121j35430y3u3l3L/icorrect-link.png](https://s3.amazonaws.com/f.cl.ly/items/430H121j35430y3u3l3L/icorrect-link.png)

Here's the link that it outputs:

```
http://aj.sandbox.wvu.edu/firstname.lastname@mail.wvu.edu
```

## Fixes

  1. Add a new field for Email in the Link modal. The only thing this would do would be to add `mailto:` in front of the email address in the link.
  1. OR automatically detect whether someone is using an email address.

_Copied from original issue: wvuweb/cleanslate#53_

**Comment by adamjohnson** [comment]
_From @dustinmazon on August 12, 2014 20:0_

I second this request. Auto detection would be a nice feature, as well as the option to create an email link.

**Comment by adamjohnson** [comment]
_From @nreckart on August 15, 2014 17:13_

The 'Standard Link' field will now automatically detect email addresses and prepend 'mailto' when creating the link. All of the following types of values should now work for the field.

* http://google.com
* google.com
* foo.bar@foobar.com
* mailto:janedoe@fake.com

This has been deployed to production.

**Comment by adamjohnson** [comment]
Dude, killing it. :thumbsup: :thumbsup: :thumbsup:

**Comment by adamjohnson** [comment]
Implemented in b61246fef7d6b5ae67751451bc57eabe3d10ffa9.
